Thorium-Sim/thorium-nova
# Flight Director Starmap systems

## Description

Makes it possible to view systems and their contents on the flight director starmap. There's still a lot of polish to be done, but this is a good first pass.

## Related Issue








Closes #285 
Closes #287 

## How do you know the changes work correctly?






Manual testing
- Start a flight
- Open the fight director screen
- Open the starmap core
- Double-click on a system
- See the entities inside the system

## Screenshots (if appropriate):
<img width="878" alt="Screen Shot 2022-06-24 at 8 01 17 PM" src="https://user-images.githubusercontent.com/6558157/175754084-a91cfdec-99e0-476e-8cdf-14d3250106fe.png">

## Types of changes



      change)

## Checklist:
